sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -621,13 +621,44 @@ void gnrc_rpl_recv_DIO(gnrc_rpl_dio_t *dio, ipv6_addr_t *src, uint16_t len)
     }
 }
 
-void _dao_fill_target(gnrc_rpl_opt_target_t *target, ipv6_addr_t *addr)
+gnrc_pktsnip_t *_dao_fill_target(gnrc_pktsnip_t *pkt, ipv6_addr_t *addr)
```

Comment:
Optional, but if you want to keep in linne with the rest of `gnrc`, this function would be called `_dao_target_build()` ;-)